src/js/generalFunctions.js

```diff
@@ -0,0 +1,7 @@
+function generateDomElement(currentNode, element, ...classes) {
```

Comment:
why do you not use the first argument and overwrite immediately?

Reply:
Done